code/modules/mob/living/carbon/monkey/life.dm

```diff
@@ -750,6 +750,23 @@
 			emote("scratch")
 			return
 
+/**
+ * Returns a number between -2 to 2.
+ * TODO: What's the default return value?
```

Comment:
I was going to say something here, based on our defines in the code, but this is all copy pasted from the human implementation of this check, lol